db/controllers/controller.js

```diff
@@ -7,8 +7,28 @@ const {
   postCommentToDatabase,
   checkUsernameExists,
   updateArticleVotes,
+  deleteCommentFromDatabase,
 } = require("../models/model");
 
+const deleteComment = (req, res, next) => {
+  const { comment_id } = req.params;
+  deleteCommentFromDatabase(comment_id)
+    .then(() => {
+      res.status(204).send({});
+    })
+    .catch((error) => {
+      if (error.status === 404) {
```

Comment:
Rather than sending a status here, it would be preferable to pipe them into you error handling middleware chain